tests/functional/lvmo/test_lvm_clone_base.py

```diff
@@ -8,6 +8,7 @@
 )
 from ocs_ci.framework.testlib import skipif_ocs_version, ManageTest, acceptance
 from ocs_ci.ocs import constants
+from ocs_ci.utility.utils import get_ocp_version
```

Comment:
there is no use for `ocp_version` in this test. these lines removed by another PR not long ago